src/requirements.txt

```diff
@@ -18,11 +18,12 @@ openai==1.13.3
 
 # azure dependencies
 azure-core==1.30.1
+azure-mgmt-authorization==4.0.0
```

Comment:
codespace creation uses requirements.freeze.txt. I'll try to make sure adding this wont break the deps as we have a ton of them.

Reply:
nvm, Just checked azure-mgmt-authorization is required by azureml-rag and we have it installed already.